napari/plugins/hub.py

```diff
@@ -44,6 +44,18 @@ def hub_plugin_info(
     except error.HTTPError:
         return None, False
 
+    # If the napari hub returns an empty info dict,
+    # simply return None, False like the above except
+    if info == {
+        'conda': [],
+        'display_name': '',
+        'plugin_types': [],
+        'reader_file_extensions': [],
+        'writer_file_extensions': [],
+        'writer_save_layers': [],
+    }:
+        return None, False
```

Comment:
I do not like this. The conda response may change in the future, and this check will be wrong. I would prefer to return these values if, for example `"version"` is missed from dict.

Reply:
added a suggestion